static/templates/add_poll_modal.hbs

```diff
@@ -0,0 +1,7 @@
+<form id="add-poll-form" class="new-style">
+    <label>Question</label>
+    <textarea id="poll-question-textarea" placeholder="{{t 'Add question'}}"></textarea>
+    <label>Options</label>
+    <p>Anyone can add more options after the poll is posted.</p>
```

Comment:
These strings all need to be tagged for translation. See

https://zulip.readthedocs.io/en/latest/translating/internationalization.html